### ISSUE TITLE
About selecting multiple tabs at once

### ISSUE THREAD

**Issue by aluncstokes** [onaralili/SplitUp#57]
This one may not even fall under the umbrella of issue, but I've so far been unable to figure out how/if it is possible (like with shift+click) instead of (CTRL+click)  to choose a range of tabs at once, or if I'm just missing something a bit obvious. Either way, very well put together extension.

Thanks!
Alun Stokes

**Comment by onaralili** [comment]
You are right! It's not possible to select range with a keyboard shortcut. This is something i can add in the next release.

**Comment by aluncstokes** [comment]
I'm slightly glad -- I was worried I was missing something bleeding obvious :P. Thanks!

**Comment by Flipmode11** [comment]
Was this ever implemented? Or at least via keyboard instead of mouse?
